### PR TITLE
Add support for Xenium 1.3 outputs

### LINKED ISSUE #77: Support xenium version 1.3
And apply to Xenium sampl, human brain datasets

### PR DESCRIPTION
**Pull request by prete**
Starting on v1.3 cell_id looks like "aaabinlp-1". Changes needed to re-index cells and match them between modalities (h5ad and label image). `pd.Categorical.codes` was used convert cell_ids to int.

### CODE REVIEW

File: bin/process_xenium.py [hunk 1]
```diff
@@ -61,39 +61,44 @@ def xenium_to_anndata(
...
 
-            clusters = cluster_df.reindex(adata.obs.index.astype(int))
-            adata.obs[cluster_name] = pd.Categorical(clusters["Cluster"])
+            clusters = cluster_df.reindex(adata.obs.index, fill_value="Undefined")
+            adata.obs[cluster_name] = pd.Categorical(clusters["Cluster"].astype(str))
 
     if load_embeddings:
         embeddings = [
             ("umap", "2_components"),
-            ("tsne", "2_components"),
```
Comment (dannda) [inline bin/process_xenium.py:78]: Is this no longer an expected output or is it just not that useful?

**Review by dannda — APPROVED** [review]

File: bin/process_xenium.py (hunk 1 above)
Comment (prete) [reply bin/process_xenium.py:78]: Both:
a) tSNE is not shown on our local Xenium outpus or the Example brain data ( both being xenium v1.3 )
b) tSNE and PCA aren't really useful in these cases as most people will only expect UMAP

**Comment by BioinfoTongLI** [comment]
I am just trying to visualize both converted version locally. will merged once confirmed

**Review by BioinfoTongLI — APPROVED** [review]
looks good to me. Test with Xenium analysis version `1.1` and `1.3`. Both loaded correctly on WebAtlas.

**Pull request merged**